src/Options.js

```diff
@@ -5,6 +5,25 @@
  * `.reset` functions.
  */
 
+const Style = require("./Style");
+
+const sizeStyleMap = [ // [textsize, scriptsize, scriptscriptsize]
```

Comment:
So these numbers are chosen by applying 0.7x and 0.5x the base size and rounding to an available KaTeX size. But your question inspired an experiment.

In old KaTeX, scriptsize is always 0.7x the size of textsize, and scriptscriptsize is 0.5x the size of textsize. This also holds in LaTeX at 10pt:

```
\documentclass[10pt]{article}
\usepackage[utf8]{inputenc}
\showboxbreadth=\maxdimen
\showboxdepth=\maxdimen
\begin{document}
\tracingonline=1
$a^{b^c}$\showlists
\end{document}
```

shows

```
\OML/cmm/m/it/10 a
\hbox(5.17056+0.0)x7.7638, shifted -3.62892
.\OML/cmm/m/it/7 b
.\hbox(2.15277+0.0)x3.74713, shifted -3.01779
..\OML/cmm/m/it/5 c
```

But at other point sizes these ratios don't hold precisely, since LaTeX was originally built for a fixed palette of fonts at specific sizes. For instance, `11pt` gives sizes 10.95/8/6, and `12pt` gives 12/8/6.

FWIW here are the sizes LaTeX uses at base font size 10pt, compared with the PR's current sizes:

TeX command | TeX Sizes | Current PR sizes
-----------|----------|------------
`\tiny` | 5/5/5 | 5/5/5
`\scriptsize` | 7/5/5 | 7/5/5
`\footnotesize` | 8/6/5 | 8/7/5 [NB KaTeX has no 6pt]
`\small` | 9/6/5 | 9/7/5
`\normalsize` | 10/7/5 | 10/7/5
`\large` | 12/8/6 | 12/8/7
`\Large` | 14.4/10/7 | 14.4/10/7
`\LARGE` | 17.28/12/10 | 17.3/12/9
`\huge` | 20.74/14.4/12 | 20.7/14.4/10
`\Huge` | 24.88/20.74/17.28 | 24.9/17.3/12

So some differences.

So what to do? We could keep the current sizes (& maybe document them). Or we could implement these more TeX-precise sizes in KaTeX, with or without the currently-missing 6pt font. We can also delay this choice (i.e. leave the current sizes in and then change them later).